week1/excercise-4/ex4.cs

```diff
@@ -0,0 +1,24 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+int Fibonacci(int n)
+    {
+       if (n <= 1)
+        {
+
+            return n;
+        }
+        else
+        {
+            return Fibonacci(n - 1) + Fibonacci(n - 2);
+        }
+                   
+       
+    }
+  
+    
+        int n = 6;
```

Comment:
Formating is a bit off, you can format the whole file by pressing ALT+SHIFT+F.

Reply:
Thanks I will update it.